src/views/Workflows.vue

```diff
@@ -0,0 +1,241 @@
+    case 'polkadot':
+      return 'https://cryptologos.cc/logos/polkadot-new-dot-logo.png';
+    case 'kusama':
+      return 'https://www.liblogo.com/img-logo/ku2766k057-kusama-logo-kusama-ksm-bitprime.png';
+  }
+}
+
+function deleteRow(rowData) {
+  message.info(`Workflow ${rowData.name} has been deleted`);
+}
+
+function handleUpdateValue(status, loading) {
+  loading = true;
+}
+
+const columns = ref([
+  {
+    title: 'Name',
+    key: 'name',
+    ellipsis: { tooltip: true },
+    className: 'text-bold',
+    sorter: true,
+    sortOrder: false,
+    renderSorterIcon: useRenderSortIcon,
+  },
+  {
+    title: 'Chain',
+    key: 'chain',
+    width: 180,
+    sorter: true,
+    sortOrder: false,
+    renderSorterIcon: useRenderSortIcon,
+    render: ({ chain }) => {
+      return h('div', { style: { display: 'flex', alignItems: 'center' } }, [
+        h(NAvatar, {
+          src: getChainImg(chain.name),
+          round: true,
+          size: 'small',
+          color: 'white',
+        }),
+        h(
+          'div',
+          { style: { marginLeft: '12px', padding: '4px 0' } },
+          chain.name
+        ),
+      ]);
+    },
+  },
+  {
+    title: 'Created at',
+    key: 'created_at',
+    ellipsis: { tooltip: true },
+    sorter: true,
+    sortOrder: false,
+    renderSorterIcon: useRenderSortIcon,
+    render: ({ created_at }) => {
+      return moment(created_at).format('MMM Do YYYY, HH:mm:ss');
+    },
+  },
+  {
+    title: 'Updated at',
+    key: 'updated_at',
+    ellipsis: { tooltip: true },
+    sorter: true,
+    sortOrder: false,
+    renderSorterIcon: useRenderSortIcon,
+    render: ({ updated_at }) => {
+      return moment(updated_at).format('MMM Do YYYY, HH:mm:ss');
+    },
+  },
+  {
+    title: 'Status',
+    key: 'status',
+    sorter: false,
+    width: '10%',
+    render: ({ status }) => {
+      return h(
+        NTooltip,
+        { showArrow: false },
+        {
+          trigger: () =>
+            // tach thanh component, refer ShowOrEdit
+            // & easier to manage loading state.
+            h(NSwitch, {
+              // loading: false,
+              // value: status,
+              onUpdateValue: (newValue) => {
+                console.log('newValue', newValue);
+              },
+            }),
+          default: () => status,
+        }
+      );
+    },
+  },
+  {
+    key: 'edit',
+    align: 'right',
+    width: '5%',
+    render(row) {
+      return h('div', [
+        h(
+          ButtonWithPopConfirm,
+          {
+            style: { 'margin-right': '1rem' },
+            onPositiveClick: () => deleteRow(row),
+            elementKey: row.id,
+            tooltipText: 'Delete workflow',
+            confirmText: 'Are you sure to delete this workflow?',
+          },
+          {
+            'trigger-content': () => h(Icon, { icon: 'bi:trash' }),
+          }
+        ),
+      ]);
+    },
+  },
+]);
+
+const createData = () => [
```

Comment:
@anhthichieu can you please move it to mock file and import it from store/api services?